boilerplates/be-docker-plain/init.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -eux
+
+# Get directory of this script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+while [[ "$#" > 0 ]]; do case $1 in
+  -p=*|--project=*) PROJECT="${1#*=}";;
+  -p|--project) PROJECT="$2"; shift;;
+
+  -c=*|--component=*) COMPONENT="${1#*=}";;
+  -c|--component) COMPONENT="$2"; shift;;
+
+  -g=*|--group=*) GROUP="${1#*=}";;
+  -g|--group) GROUP="$2"; shift;;
+
+  -t=*|--target-dir=*) TARGET_DIR="${1#*=}";;
+  -t|--target-dir) TARGET_DIR="$2"; shift;;
+
+  -o=*|--owner=*) OWNER="${1#*=}";;
+  -o|--owner) OWNER="$2"; shift;;
+
+  *) echo "Unknown parameter passed: $1"; exit 1;;
+esac; shift; done
+
+cd $TARGET_DIR
+
+mkdir -p $COMPONENT
+
+cd $COMPONENT
+
+sudo chown -R $OWNER .
+
+echo "fix nexus repo path"
```

Comment:
Is this needed? If not should it be removed?

Reply:
we could leave it so to show this might be a missing step once implementing a custom docker

Reply:
@henrjk  - is WHAT needed? .... just the "echo"?

Reply:
I thought perhaps having nexus support in the basic docker starter is not needed and somewhat unexpected. So that it could all be removed.

Reply:
@henrjk  - so I remove the comment then. (can you do me a for and next time be clear about what the change and WHY? :))